pilco/environments/continuous_mountaincar.py

```diff
@@ -0,0 +1,175 @@
+"""
+Our modification of the OpenAI Gym Continuous Mountain Car by Olivier Sigaud:
+https://github.com/openai/gym/blob/master/gym/envs/classic_control/continuous_mountain_car.py
+
+which was (ultimately) based on Sutton's implementation:
+http://incompleteideas.net/sutton/MountainCar/MountainCar1.cp
+"""
+
+import math
+
+import numpy as np
```

Comment:
Is there a reason for using both `math` and `numpy` utilities? If not, probably just switch to `numpy` only.

Reply:
Yeah, they were using it so i left it there. Changed now.